CodeForPoznan/codeforpoznan.pl_v3
# Modal content (fixes #113)

Complete although a lil bit messy in terms of css, so any suggestions (@Loczi94) in that regard would be great.

**Manual testing:**
1. docker up the app
1. click any of the projects
1. check if it renders proper details (use Volontulo - the rest doesn't have all the data filled yet)
1. check if all buttons work
1. check the console for errors
